src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInWidgets.ts

```diff
@@ -34,50 +35,55 @@ export const migrateCalculatedMeasuresInWidgets = (
       draft.children!.content.children,
       (widgetRecord) => {
         const serializedWidgetState: WidgetWithQueryState<
-          MdxSelect,
+          MdxSelect | MdxDrillthrough,
           "serialized"
         > = JSON.parse(widgetRecord.entry.content);
         const deserializedWidgetState = deserializeWidgetState(
           serializedWidgetState,
         );
-        if (isWidgetWithQueryState(deserializedWidgetState)) {
-          const mdx: MdxSelect | undefined = deserializedWidgetState.query.mdx;
 
-          if (!mdx) {
-            return widgetRecord;
-          }
-
-          const {
-            migratedMdx,
-            namesOfCalculatedMeasuresToMigrateInWidget,
-            cubeName,
-          } = migrateCalculatedMeasuresInMdx(
-            mdx,
-            namesOfCalculatedMeasurestoMigrate,
-            dataModel,
-          );
-
-          namesOfCalculatedMeasuresToMigrateInWidget.forEach(
-            (calculatedMeasureName) => {
-              measureToCubeMapping[calculatedMeasureName] =
-                measureToCubeMapping[calculatedMeasureName]
-                  ? _uniq([
-                      ...measureToCubeMapping[calculatedMeasureName],
-                      cubeName,
-                    ])
-                  : [cubeName];
-            },
-          );
-          const updatedWidgetState = produce(
+        if (
+          !isWidgetWithQueryState<MdxSelect | MdxDrillthrough>(
```

Comment:
In this file the diff seems big but it's just because I moved a branch to an early return.
The logic does not change more than wiring `dataModels` and `serverKey`, like in the other files.